i18n/en.json

```diff
@@ -2260,7 +2260,7 @@
   "delete_channel.cancel": "Cancel",
   "delete_channel.confirm": "Confirm ARCHIVE Channel",
   "delete_channel.del": "Archive",
-  "delete_channel.question": "This will archive the channel from the team and make its contents inaccessible for all users. \n \nAre you sure you wish to archive the **{display_name}** channel?",
+  "delete_channel.question": "This will archive the channel from the team and remove it from the user interface. A copy of the channel is kept on the server. \n \nAre you sure you wish to archive the {display_name} channel?",
```

Comment:
```suggestion
  "delete_channel.question": "This will archive the channel from the team and remove it from the user interface. Archived channels can be unarchived if needed again. \n \nAre you sure you wish to archive the {display_name} channel?",
```